build-system/tasks/karma.conf.js

```diff
@@ -100,9 +99,6 @@ module.exports = {
     stripPrefix: 'test-bin/',
   },
 
-  // TODO(rsimha, #15510): Sauce labs on Safari doesn't reliably support
```

Comment:
What about Safari on GitHub Actions? Is this what karma-safarinative-launcher fixed?

Reply:
This was an old Sauce-specific problem documented [here](https://support.saucelabs.com/hc/en-us/articles/115002212447-Unable-to-Reach-Application-on-localhost-for-Tests-Run-on-Safari-8-and-9-and-Edge), and shouldn't affect GH actions because we do not access the browser via a proxy. 

`karma-safarinative-launcher` is for a different purpose. It eliminates the blocking launch dialog described in https://github.com/karma-runner/karma-safari-launcher/issues/29.